gradle/libs.versions.toml

```diff
@@ -4,24 +4,16 @@ kotlin = "2.0.21"
 vanniktech = "0.29.0"
 ktlint = "12.1.0"
 dokka = "1.9.20"
-kotlinx_datetime = "0.6.0"
-kotlinx_coroutines = "1.8.1"
 kotlinx_serialization = "1.7.1"
-pubnub = "10.1.0"
+pubnub = "10.2.0"
 
 [libraries]
 pubnub-kotlin-api = { module = "com.pubnub:pubnub-kotlin-api", version.ref = "pubnub" }
-pubnub-kotlin-impl = { module = "com.pubnub:pubnub-kotlin-impl", version.ref = "pubnub" }
 pubnub-kotlin = { module = "com.pubnub:pubnub-kotlin", version.ref = "pubnub" }
 pubnub-kotlin-test = { module = "com.pubnub:pubnub-kotlin-test", version.ref = "pubnub" }
-#slf4j = { module = "org.slf4j:slf4j-api", version = "1.7.36" }
-#cbor = { module = "co.nstant.in:cbor", version = "0.9" }
-#jetbrains-annotations = { module = "org.jetbrains:annotations", version = "24.1.0" }
 kotlinx-atomicfu = { module = "org.jetbrains.kotlinx:atomicfu", version = "0.25.0" }
 kotlinx-serialization-core = { module = "org.jetbrains.kotlinx:kotlinx-serialization-core", version.ref = "kotlinx_serialization" }
-kotlinx-datetime = { module = "org.jetbrains.kotlinx:kotlinx-datetime", version.ref = "kotlinx_datetime"}
 touchlab-kermit = { module = "co.touchlab:kermit", version = "2.0.4" }
-diff-match-patch = { module = "org.bitbucket.cowwoc:diff-match-patch", version = "1.2" }
```

Comment:
Is was not needed even before this PR, right?

Reply:
yes, we didn't use it in the end